services/processor/native/sanitize_code.go

```diff
@@ -28,17 +28,14 @@ func SanitizerConfigForProduction() *sanitizer.SanitizerConfig {
 			// package: reason to whitelist
 
 			// SDK
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1"`:                      "SDK",
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/address"`:              "SDK",
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/env"`:                  "SDK",
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/ethereum"`:             "SDK",
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/events"`:               "SDK",
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/safemath"`:             "SDK",
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/safemath/safeuint32"`:  "SDK",
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/safemath/safeuint64"`:  "SDK",
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/safemath/safeuint256"`: "SDK",
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/service"`:              "SDK",
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/state"`:                "SDK",
+			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1"`:            "SDK",
```

Comment:
why not `"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/*`?

Reply:
changed, found another issue with usage of the root level of the wildcard. so in safemath, just importing 'safemath/' and not a sub-package
if changing it to `v1/*` then the v1 import failed (that is what the last commit fixes)